viz-lib/src/visualizations/chart/Editor/ColumnMappingSelect.jsx

```diff
@@ -12,18 +12,17 @@ const MappingTypes = {
   zVal: { label: "Color Column" },
 };
 
+const SwappedMappingTypes = {
+  ...MappingTypes,
+  x: { label: "Y Column" },
+  y: { label: "X Columns", multiple: true }
+}
+
 export default function ColumnMappingSelect({ value, availableColumns, type, onChange, areAxesSwapped }) {
   const options = sortBy(filter(uniq(flatten([availableColumns, value])), v => isString(v) && v !== ""));
 
-  let { label, multiple } = MappingTypes[type];
   // this swaps the ui, as the data will be swapped on render
-  if (areAxesSwapped) {
-    if (type === "x") {
-      label = label.replace("X", "Y");
-    } else if (type === "y") {
-      label = label.replace("Y", "X");
-    }
-  }
+  const { label, multiple } = !areAxesSwapped ? MappingTypes[type] : SwappedMappingTypes[type]
```

Comment:
Cleaner syntax, adopting @kravets-levko's suggestions. I also renamed everything. Guess I was with a narrow vocab at the time, swap is way better! Thanks!